### PR TITLE
[WIP] Blog post: Three questions for our open service

### LINKED ISSUE #473: [blog] Three open questions for our service
### Summary / idea for post

In some recent brainstorms with @malvikasharan @aldenc @harisood and @Arielle-Bennett, we came up with a few major questions to explore and answer as a part of improving our collaborative service model. This post will describe these questions briefly and note why we think they are both important and complex to answer.

This is meant to be a quick share-out from this meeting so that we are transparent about what we're thinking about, and in case others find it interesting or want to participate.

### Tasks and updates

- [ ] Finish https://github.com/2i2c-org/team-compass/issues/452 first
- [ ] Post draft: https://github.com/2i2c-org/2i2c-org.github.io/pull/104
- [ ] Blog location: {LINK} 
- [ ] Publish to blog
- [ ] Send out tweets and LinkedIn
- [ ] Send to 2i2c listserv (optional)

### PR DESCRIPTION
**Pull request by choldgraf**
This is a working draft outline that I'll turn into a blog post, see issue below for context:

closes https://github.com/2i2c-org/team-compass/issues/473

### To do

- [ ] Finish up general outline and main points
- [ ] Flesh out with content
- [ ] Feedback
- [ ] Post

### CODE REVIEW

File: content/blog/2022/turing-open-service-questions/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+featured: false
+draft: false
+---
+
+# Three open questions for building open services
+
+Over the past few weeks I've had an opportunity to brainstorm with members from [The Turing Way](https://the-turing-way.netlify.app/welcome) and [The Turing Institute](https://www.turing.ac.uk/) about how 2i2c can refine is collaborative service model to be more impactful.
+We recently had a day of brainstorming with [Malvika Sharan](https://github.com/malvikasharan), [Alden Conner](https://github.com/aldenc), [Hari Sood](https://github.com/harisood), and [Arielle Bennett](https://github.com/Arielle-Bennett).
+One product that came out of this meeting was three important questions that we need to answer.
+The reflect several major questions around creating community-centered services that strike the right division of labor and responsibility for each partner in the service.
```
Comment (damianavila) [inline content/blog/2022/turing-open-service-questions/index.md:18]: `That`, maybe?

Comment (damianavila) [reply content/blog/2022/turing-open-service-questions/index.md:18]: Or am I misunderstanding the phrase?

File: content/blog/2022/turing-open-service-questions/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Some of our discussions around this can also be found in [this GitHub Issue](https://github.com/2i2c-org/team-compass/issues/446).
+
+## Where is the line between 2i2c and communities?
+
+- We want to build spaces for communities to interact with us, but do not want communities to lose their own identity
+- Example: communication forums
+- How do we avoid colonizing them
```
Comment (damianavila) [inline content/blog/2022/turing-open-service-questions/index.md:26]: `colonizing` has a really bad connotation... maybe another word?

File: content/blog/2022/turing-open-service-questions/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- What if they don't have a pre defined community structure? Or are small? 
+- When is it time for them to move on and create their own space?
+
+## What can we ask of communities?
+
+- We treat each hub as a collaboration
+- We can provide infrastructure and guidance around it, but communities know what is best for themselves
+- What can we ask of them in collaborating with us in the service? What subsets of people could take on more work than others? Post docs? Probably not PIs
```
Comment (damianavila) [inline content/blog/2022/turing-open-service-questions/index.md:34]: You are assuming there are post-docs and PIs... maybe that is common in our communities but I can see other structures as well in other sorts of communities.

File: content/blog/2022/turing-open-service-questions/index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- How can we support them in this? Could we provide suggested structure? Training via partnerships with non profits like cscce, openscapes, etc? Financial incentive for community members to step up?
```
Comment (damianavila) [inline content/blog/2022/turing-open-service-questions/index.md:35]: Links for cscee and openscapes?

File: content/blog/2022/turing-open-service-questions/index.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+- What if they don't have a pre defined community structure? Or are small? 
+- When is it time for them to move on and create their own space?
+
+## What can we ask of communities?
+
+- We treat each hub as a collaboration
+- We can provide infrastructure and guidance around it, but communities know what is best for themselves
+- What can we ask of them in collaborating with us in the service? What subsets of people could take on more work than others? Post docs? Probably not PIs
+- How can we support them in this? Could we provide suggested structure? Training via partnerships with non profits like cscce, openscapes, etc? Financial incentive for community members to step up? 
+- What leadership is crucial that others in the community show what allowed Pangeo to thrive?
```
Comment (damianavila) [inline content/blog/2022/turing-open-service-questions/index.md:36]: More context about the successfulness of Pangeo (or a link to an article about Pangeo's success?)

**Comment by damianavila** [comment]
Since this is not in draft mode I jumped and provided feedback, @choldgraf.
Then I noticed you said at the top this is a draft.
My bad...

**Comment by choldgraf** [comment]
Haha thanks for the feedback anyway! I should have marked it as draft, sorry

**Comment by choldgraf** [comment]
Going to close this one, since I don't think we have time to finish it up and have had similar communications and posts since then. Sorry @damianavila that you reviewed this one even though it's not gonna go out :-/

see https://github.com/2i2c-org/team-compass/issues/473#issuecomment-1306834215 for linked comment

**Pull request closed without merging**